packages/backend/src/ormconfig.ts

```diff
@@ -2,26 +2,26 @@ import { sync as globSync } from 'fast-glob';
 import { join as joinPath, relative as relativePath } from 'path';
 import { ConnectionOptions } from 'typeorm';
 
-import { databaseUrl, databaseMainSchema, root, typeormLogging } from './config/config';
+import { databaseUrl, root, typeormLogging } from './config/config';
 
 const connectionOptions: ConnectionOptions = {
-  type: 'postgres',
-  url: databaseUrl,
-  schema: databaseMainSchema,
-  synchronize: false,
-  logging: typeormLogging,
-  entities: findAppFiles('entity/*.[jt]s'),
-  migrations: findAppFiles('migration/*.[jt]s'),
-  subscribers: findAppFiles('subscriber/*.[jt]s'),
-  cli: {
-    entitiesDir: relativePath(root, joinPath(__dirname, 'entity')),
-    migrationsDir: relativePath(root, joinPath(__dirname, 'migration')),
-    subscribersDir: relativePath(root, joinPath(__dirname, 'subscriber'))
-  }
+    type: 'postgres',
+    url: databaseUrl,
+    schema: 'modos',
```

Comment:
Because it is hardcoded in the migrations. It's useless to let the user configure it if it makes the migrations fail. We could interpolate the variable into the migrations, but I don't think we need to spend the time.